test/integration/targets/aws_s3_bucket_notification/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
```

Comment:
Need to make the test suite name match the module name. I don't mind if that's adding `aws_` to the module name or removing `aws_` from the test suite name

Reply:
Removing `_aws` from the test suite name seems better to me.